plugins/zstack.py

```diff
@@ -125,10 +125,12 @@ def _estimate_step_duration(self):
         return (float > 0): estimated time (in s) that it takes to move the focus
           by one step.
         """
+        speed = None
         if model.hasVA(self.focus, "speed"):
-            speed = self.focus.speed.value['z']
-        else:
+            speed = self.focus.speed.value.get('z', None)
+        if speed is None:
```

Comment:
Maybe it could be logged that None is replaced by a (very small) number?

Reply:
That's only used for the estimated acquisition time. I don't think it's a big deal that we need to put log messages.